roles/reverseproxy/tasks/main.yml

```
@@ -0,0 +1,8 @@
+---
+- name: server configuration
+  include_tasks: server.yml
+  when: reverseproxy_server is true
+
+- name: client configuration
+  include_tasks: client.yml
+  when: reverseproxy_server is false
```

```suggestion
  when: not reverseproxy_server
```

Might be personal preference though